dbt-get-started/README.md

```diff
@@ -26,6 +26,9 @@ To access the [dbt CLI](https://docs.getdbt.com/dbt-cli/cli-overview), run:
 
 ```bash
 docker exec -it dbt bash
+
+#TODO: the dbt-materialize:v0.26.1 release doesnt have the latest dbt test code. Use the newest image and remove this line on the next release.
+pip install dbt-materialize --upgrade
```

Comment:
This should be transparent to the user, so if the image doesn't have the latest version of the adapter, we should build out of a Dockerfile with this baked in instead.

Reply:
Ok! Made a dockerfile for now while we figure the dbt image update.